src/Api/Objects/Eet.php

```diff
@@ -34,6 +34,12 @@ class Eet extends AbstractObject
 	/** @var string */
 	public $currency;
 
+	/** @var float|null */
+	public $subsequentDrawing;
+
+    /** @var float|null */
```

Comment:
```suggestion
	/** @var float|null */
```

prosím i o opravu mezery -> taby v src/Api/Entity/PaymentFactory.php